pom.xml

```diff
@@ -47,9 +47,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- versions -->
-    <version.io.opentracing-opentracing>0.32.0</version.io.opentracing-opentracing>
-    <version.io.opentracing.contrib-opentracing-concurrent>0.3.0</version.io.opentracing.contrib-opentracing-concurrent>
-    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.3.0</version.io.opentracing.contrib-opentracing-web-servlet-filter>
+    <version.io.opentracing-opentracing>0.33.0</version.io.opentracing-opentracing>
+    <version.io.opentracing.contrib-opentracing-concurrent>0.4.0</version.io.opentracing.contrib-opentracing-concurrent>
```

Comment:
These just have been released. It might take some time to propagate to central

Reply:
@pavolloffay Any chance for merging/releasing this PR?  It looks like all required dependencies are already in central. Thanks

Reply:
@kasinskim four curiosity what client are you using?

Reply:
I'm at the very beginning of tracing journey. Currently piloting `jaeger` implementation (for both Java/Node technologies).

BTW. Do you have any plans on providing instrumentation for https://github.com/open-telemetry?